CONTRIBUTING.md

```diff
@@ -298,6 +298,32 @@ Speaking of tests, Rust has a comprehensive test suite. More information about
 it can be found
 [here](https://github.com/rust-lang/rust-wiki-backup/blob/master/Note-testsuite.md).
 
+### External Dependencies
+
+Currently building Rust will also build the following external projects:
```

Comment:
I think this documentation may wish to be updated with the current state of the PR